Code/Legacy/CrySystem/XConsole.cpp

```diff
@@ -1682,11 +1682,12 @@ void CXConsole::DisplayHelp(const char* help, const char* name)
     else
     {
         char* start, * pos;
-        for (pos = strstr((char*)help, "\n"), start = (char*)help; pos; start = ++pos)
+        for (pos = strstr((char*)help, "\n"), start = (char*)help; pos; )
         {
             AZStd::string s = start;
             s.resize(pos - start);
             ConsoleLogInputResponse("    $3%s", s.c_str());
+            start = ++pos;
```

Comment:
This can be changed to a call to `AZ::StringFunc::TokenizeVisitor`
```
auto DisplayLines [](AZStd::string_view token)
{
    AZStd::fixed_string<256> lineStorage(token);
    ConsoleLogInputResponse(" $3%s", lineStorage.c_str());
};
AZ::StringFunc::TokenizeVisitor(help, DisplayLines, '\n');
```